feature-libs/organization/administration/components/unit/links/users/list/unit-user-link-cell.component.ts

```diff
@@ -32,11 +33,12 @@ export class UnitUserRolesCellComponent extends CellComponent {
   unitKey$: Observable<string> = this.itemService.key$;
   constructor(
     protected outlet: OutletContextData<TableDataOutletContext>,
-    protected itemService: ItemService<B2BUnit>
+    protected itemService: ItemService<B2BUnit>,
+    protected b2bUserService: B2BUserService
   ) {
     super(outlet);
   }
-
+  isUpdatingUserAllowed = this.b2bUserService.isUpdatingUserAllowed();
```

Comment:
In my opinion `b2bUserService` argument should be optional. And if service is not provided the `isUpdatingUserAllowed` should return `true`, to avoid mis-rendering template elements on each the value of `isUpdatingUserAllowed` has been used in `*ngIf` directive.

Reply:
done